sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-institution-designation-report.sql

```diff
@@ -0,0 +1,41 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Institution_Designation_Report',
+    'SELECT
+      institutions.operating_name AS "Institution Operating Name",
```

Comment:
Minor, are the other SQL files using "AS" explicitly?

Reply:
Yes, they are.